web/src/components/tasks/business-formation/contacts/ContactsStep.tsx

```diff
@@ -62,7 +70,7 @@ export const ContactsStep = (): ReactElement => {
             {getErrorStateForField({ field: "signers", formationFormData: state.formationFormData }).label}
           </Alert>
         )}
-        {doesFieldHaveError("incorporators") && (
+        {doesFieldHaveError("incorporators") && !isCorpAndLp() && (
```

Comment:
The `!isCorpAndLp` condition is necessary so that this alert doesn't render for domestic corporations (s & c) and LPs. Otherwise, the banner alert will populate and we now want an inline instead for these legal structures.